regions/src/main/java/software/amazon/awssdk/regions/util/EC2MetadataUtils.java

```diff
@@ -60,6 +61,7 @@
  * EC2 User Guide: Instance Metadata</a>
  */
 @ReviewBeforeRelease("Cleanup")
+@SdkInternalApi
```

Comment:
This could be useful to customers

Reply:
Before GA, sure. It's not ready to be public right now, though.